src/utils/source-maps.js

```diff
@@ -33,7 +33,12 @@ export async function getGeneratedLocation(
   return {
     line,
     sourceId,
-    column: column === 0 ? undefined : column,
+    // If the input location didn't have a column number, and we got back
+    // the start of the line, we consider the result to also not have a column.
+    // This is so that we can preserve our current behavior of having
+    // breakpoints with optional columns, while allowing operations on
+    // locations unrelated to breakpoints to always have a column.
+    column: location.column === undefined && column === 0 ? undefined : column,
```

Comment:
is it fair to change `getGeneratedLocation` to always have a column and assume we get a good column from  location?

Reply:
Hmm, I think if we are concerned about things breaking and want it behind a pref, this is definitely one we'd want behind a pref. My intention with this is that the breakpoint pref would leave `column: undefined` and therefore propagate through into here, but if the pref for default-0 is one, we'll always get a column out of this too.